src/GridList.jsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import cx from 'classnames';
+
+/**
+ * @module GridList
+ */
+class GridList extends React.Component {
+	renderItems() {
+		this.gridListItems = this.props.items.map((item, i) => {
+			return (
+				<div
+					key={i}
+					className='gridList-item'
+				>
+					{item}
+				</div>
+			);
+		});
+		return this.gridListItems;
```

Comment:
given how small the `render` method is for this component, I think it would be more direct to just put this `renderItems` method inside `render` directly rather than requiring engineers to jump up to this method when debugging a render error. The whole JSX block in `render` could be tightened up without obscuring things too much:

```jsx
render() {
  // ...
  return (
    <ul className={classNames} {...other}>
      {items.map((item, key) =>
          <div key={key} className='gridList-item'>{item}</div>
      )}
    </ul>
  );
}
```